src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -2,7 +2,6 @@ package cromwell.logging
 
 import java.text.SimpleDateFormat
 import java.util.Date
-
```

Comment:
I don't think this change is necessary, and the default IntelliJ formatter is just going to add it back. :neutral_face: